src/Masonry/index.js

```diff
@@ -7,6 +7,7 @@ class Masonry extends React.Component {
     const columns = Array.from({length: columnsCount}, () => [])
 
     React.Children.forEach(children, (child, index) => {
+      if(typeof child !== "number" && !child) return null
```

Comment:
why `typeof child !== "number"`?

`if(!child) return null` is not enough?

Reply:
hey @cedricdelpoux,
By using `typeof child !== "number"`  it will allow `0` as valid child, else `number 0` will get neglected.

Reply:
Maybe you can use [isValidElement](https://reactjs.org/docs/react-api.html#isvalidelement) or something similar. I find it more explicit